### PR TITLE
[ADD] Account-PT

### PR DESCRIPTION
**Pull request by thinkopen-pt**
This module extends the base Account with new versions of the invoices (incl. simplified invoices and debit notes), waybills, and adapts the methods that create invoices from sale orders, stock pickings, etc to the these changes.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
You can't put your company initials on the OCA modules

**Comment by max3903** [comment]
and you need to follow the guidelines:
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.rst

which mentions using the README template from:
https://github.com/OCA/maintainer-tools/tree/master/template/module

**Comment by pedrobaeza** [comment]
This modules is also a very bloated module with a lot of functions. You should split it in smaller and more moduler ones.

RML reports are also deprecated, but for now, it's enough.

File: tko_account_pt/__init__.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- encoding: utf-8 -*-
```
Comment (dreispt) [inline tko_account_pt/__init__.py:1]: Rename module to `l10n_pt_account`

Comment (dreispt) [inline tko_account_pt/__init__.py:1]: I think it's `coding: utf-8`

File: tko_account_pt/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Contabilidade PT',
+    'version': '1.122',
+    'author': 'ThinkOpen Solutions,Odoo Community Association (OCA)',
+    'license': 'Other OSI approved licence',
```
Comment (yvaucher) [inline tko_account_pt/__openerp__.py:31]: Please specify the right license, as far as I see in you headers, it is GPL and GPL v3 or are you planning to set it as LGPL waiting for the key to be valid https://github.com/odoo/odoo/pull/6914?

File: tko_account_pt/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        'simplified_invoice_view.xml',
+        'sale_view.xml',
+        'account_config_view.xml',
+        'res_config_view.xml',
+    ],
+    'test': [],
+    'installable': True,
+}
+
+# vim:expandtab:smartindent:tabstop=4:softtabstop=4:shiftwidth=4:
```
Comment (yvaucher) [inline tko_account_pt/__openerp__.py:68]: You can get ride of this line, all vim users I know don't rely on it for Odoo modules

File: tko_account_pt/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Contabilidade PT',
```
Comment (yvaucher) [inline tko_account_pt/__openerp__.py:28]: If you want, you can add a `summary` with a oneline short description for this module, displayed on apps and in Odoo module list.

File: tko_account_pt/__openerp__.py (hunk 2 above)
Comment (thinkopen-pt) [inline tko_account_pt/__openerp__.py:31]: @yvaucher Yes, we're planning to set it as LGPL.

Comment (yvaucher) [inline tko_account_pt/__openerp__.py:31]: Maybe you can already change your license headers then.

**Pull request closed without merging**